resources/js/Components/Form/DmEntryCreateForm.tsx

```diff
@@ -0,0 +1,479 @@
+import {Link, useForm, usePage} from '@inertiajs/inertia-react'
```

Comment:
we have a new `Link` component in atom folder, use that otherwise it will have no styling